api/swagger.yaml

```diff
@@ -1975,11 +1975,37 @@ definitions:
     description: "User modifiable task configuration."
     type: "object"
     properties:
+      PluginSpec:
+        type: "object"
+        properties:
+          Name:
+            description: "The name or 'alias' to use for the plugin. Must not specify with `ContainerSpec`"
+            type: "string"
+          Remote:
+            description: "The plugin image reference to use."
+            type: "string"
+          Disabled:
+            description: "Disable the plugin once scheduled."
+            type: "boolean"
+          PluginPrivilege:
+            type: "array"
+            items:
+              description: "Describes a permission accepted by the user upon installing the plugin."
+              type: "object"
+              properties:
+                Name:
+                  type: "string"
+                Description:
+                  type: "string"
+                Value:
+                  type: "array"
+                  items:
+                    type: "string"
       ContainerSpec:
         type: "object"
         properties:
           Image:
-            description: "The image name to use for the container."
+            description: "The image name to use for the container. Must not specify with `PluginSpec`"
```

Comment:
Not sure if this comment is obvious. "Must not specify" means: "Not a valid parameter with` ?